book3/14-objects.mkd

```diff
@@ -107,123 +110,121 @@ looking at the output of the `dir()` function:
+leerlo, para asegurarte de haberlo comprendido correctamente.
 
-Starting with programs
+Comenzando con programas
 ----------------------
 
-A program in its most basic form takes some input, does some processing, and
-produces some output.  Our elevator conversion program demonstrates a very
-short but complete program showing all three of these steps.
+En su forma más básica, un programa toma algún dato de entrada, lo procesa
+y luego produce un resultado. Nuestro programa de conversión de un elevador
+muestra una manera muy corta, pero completa, de llevar a cabo estos tres pasos.
 
 \VerbatimInput{../code3/elev.py}
 
-If we think a bit more about this program, there is the "outside world" and
-the program. The input and output aspects are where the program interacts
-with the outside world.  Within the program we have code and data to accomplish
-the task the program is designed to solve.
+Si pensamos un poco más sobre este programa, existe un "mundo exterior" y
+el programa. Es con los datos de entrada y de salida que el programa interactúa
+con el mundo exterior. Dentro del programa utilizamos tanto el código como los datos
+para cumplir la tarea que el programa está diseñado para resolver.
 
 ![A Program](height=1.20in@../images/program)
 
-One way to think about object-oriented programming is that it separates
-our program into multiple "zones." Each zone contains some code and data
-(like a program) and has well defined interactions with the outside world
-and the other zones within the program.
+Una forma de pensar en la programación orientada a objetos es que separa
+nuestro programa en varias "zonas". Cada zona contiene algo de código
+y datos (como un programa) y tiene interacciones bien definidas tanto
+con el mundo exterior como con las otras zonas del programa.
 
-If we look back at the link extraction application where we used the
-BeautifulSoup library, we can see a program that is constructed by connecting
-different objects together to accomplish a task:
+Si miramos la aplicación de extracción de enlaces en la que usamos
+la librería BeautifulSoup, podemos ver un programa que fue construido
+conectando distintos objetos para cumplir una tarea:
 
 \index{BeautifulSoup}
 \index{HTML}
 \index{parsing!HTML}
 
 \VerbatimInput{../code3/urllinks.py}
 
-We read the URL into a string and then pass that into `urllib` to retrieve
-the data from the web.  The `urllib` library uses the `socket` library to
-make the actual network connection to retrieve the data. We take the string
-that `urllib` returns and hand it to BeautifulSoup for parsing.
-BeautifulSoup makes use of the object
-`html.parser`^[https://docs.python.org/3/library/html.parser.html] and returns an object.
-We call the `tags()` method on the returned object that returns a dictionary of tag
-objects. We loop through the tags and call the `get()` method for each tag
-to print out the `href` attribute.
+Convertimos la URL en una cadena y luego pasamos a ésta por 'urllib' para
```

Comment:
Use backticks here, for the urllib and socket words in this paragraph. Also for html.parser in line 152.